project/BRANCHES-AND-TAGS.md

```diff
@@ -13,16 +13,29 @@ Keep in mind that release branches only accept bug and security fixes; new featu
+|-----------------------------|------------------------------------------------|-----------------------|-----------------------------|-------------------------------------------|
+| master (development branch) | The Moby Project [MAINTAINERS](../MAINTAINERS) | N/A                   | -                           | N/A                                       |
+| 27.x                        | The Moby Project [MAINTAINERS](../MAINTAINERS) | Maintained            | After 28.x                  | [Docker, Inc.][docker], [Microsoft][msft] |
+| 26.1                        |                                                | Unmaintained          |                             |                                           |
+| 26.0                        |                                                | Unmaintained          |                             |                                           |
+| 25.0                        | @corhere @austinvazquez                        | Maintained            | TBD                         | [Amazon][al2023], [Mirantis][mcr]         |
+| 24.0                        |                                                | Unmaintained          |                             |                                           |
+| 23.0                        | @corhere                                       | Maintained (security) | [2025-05-19][mcr23-maint]   | [Mirantis][mcr]                           |
+| 22.06                       |                                                | Unmaintained          |                             |                                           |
+| 20.10                       |                                                | Unmaintained          |                             |                                           |
+| 19.03                       |                                                | Unmaintained          |                             |                                           |
+| 17.05.x                     |                                                | Unmaintained          |                             |                                           |
+| 17.04.x                     |                                                | Unmaintained          |                             |                                           |
+| 17.03.x                     |                                                | Unmaintained          |                             |                                           |
+| 1.13.x                      |                                                | Unmaintained          |                             |                                           |
+| 1.12.x                      |                                                | Unmaintained          |                             |                                           |
```

Comment:
BIt longer now, but added all the branches. We could move the unmaintained ones to a separate table at some point I guess

Reply:
TBH, I think the previous was clearer 😅 
Or if we want to keep a longer list, at least don't include older than 20.10?

Reply:
Maybe keep the unmaintained ones newer than the oldest maintained one (so, 26.1, 26.0 and 24.0), and then just put "Older than 23.0 - Unmaintained" in the bottom row?

Reply:
Yes, perhaps that works. I was looking at the branches and thought possibly it was confusing if `25.0` and `27.x` were maintained, but branches in between missing from the list. I also considered _removing_ the `26.1` branch from the table and only keep the maintained branches in it, but as it's a recent change, I thought it would be good to keep it to make it more visible "this branch is no longer maintained".


Let me update.